00-HTML-CSS-basics/HTMLExercises/media.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <style>
+        body{
+            margin: 0;
+            padding: 0;
+        }
+        h1{
+            padding: 10px;
+            margin: 0;
+        }
+        header{
+            text-align: center;
+            background-color: rgb(83, 65, 83);
+            color: oldlace;
+        }
+        #container{
+            margin: 10px;
+            width: 100%;
+            height: 100%;
+            text-align: center;
+        }
+        .image, .audio, .video{
+            display: inline-block;
+            width: 30%;
+        }
+        #idAudio, #idVideo{
+            height: 250px;
+            width: 96%;
+        }
+    </style>
+</head>
+<body>
```

Comment:
The idea here is to use HTML5 tags (Check the support for multimedia content here: http://diveintohtml5.info/). Also keep in mind the possibility that the loading of the resource could fail and you have to still informing the user that there some content exist. Please review this exercise and redo.